scripts/tasks/webpack-resources.js

```diff
@@ -90,7 +90,7 @@ module.exports = {
           extensions: ['.ts', '.tsx', '.js']
         },
 
-        devtool: 'eval',
+        devtool: 'source-map',
```

Comment:
This will slow innerloop down pretty significantly. For the purpose of the doing iterative development, eval is pretty good when combined with using the sourcemap support it offers.

Reply:
This is for debugging and I didn't mean to push it. But yeah, for sure, it needs to be reverted.

Reply:
Okay this is the only thing that blocks me from approving this change. Once you revert, I'll approve